sdk/identity/Azure.Identity/src/Credentials/DefaultAzureCredentialOptions.cs

```diff
@@ -124,10 +126,12 @@ public string SharedTokenCacheTenantId
         }
 
         /// <summary>
-        /// The tenant id of the user to authenticate, in the case the <see cref="DefaultAzureCredential"/> authenticates through, the
+        /// The tenant ID of the user to authenticate, in the case the <see cref="DefaultAzureCredential"/> authenticates through, the
         /// <see cref="VisualStudioCredential"/>. The default is null and will authenticate users to their default tenant.
-        /// The value can also be set by setting the environment variable AZURE_TENANT_ID.
         /// </summary>
+        /// <remarks>
+        /// This value can alternatively be set via environment variable <c>AZURE_TENANT_ID</c>.
+        /// </remarks>
```

Comment:
Do we want to add this for the EBN'd properties? The recommended property to utilize for all of these more specifiic TenantId properties is `public string TenantId`. Looks like we are missing the blurb about the environment variable option there.

Reply:
I reverted changes to the EBN'd properties, since they don't appear in IntelliSense or in the Learn API ref pages.